docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -304,6 +304,9 @@ Noteworthy module changes
 * ``vmware_vm_facts`` used to return dict of dict with virtual machine's facts. Ansible 2.8 and onwards will return list of dict with virtual machine's facts.
   Please see module ``vmware_vm_facts`` documentation for example.
 
+* ``vmware_guest_snapshots`` module used to return result. Since Ansible 2.8 and onwards result is replaced by snapshot_results.
```

Comment:
```suggestion
* ``vmware_guest_snapshot`` module used to return ``results``. Since Ansible 2.8 and onwards ``results`` is a reserved keyword, it is replaced by ``snapshot_results``.
```